astropy/cosmology/_src/traits/hubble.py

```diff
@@ -38,3 +44,18 @@ def hubble_parameter(self, z: Quantity | ArrayLike) -> Quantity:
             Hubble parameter at each input redshift.
         """
         return self.H0 * self.efunc(z)
+
+    @cached_property
+    def h(self) -> float:
+        """Dimensionless Hubble constant: h = H_0 / 100 [km/sec/Mpc]."""
+        return self.H0.value / 100.0
+
+    @cached_property
+    def hubble_time(self) -> u.Quantity:
+        """Hubble time."""
+        return (1 / self.H0).to(u.Gyr)
```

Comment:
I like the simplification! The `_sec_to_Gyr` and `_H0units_to_invs` can be removed from above.